test/tests/functional/pbs_mom_hook_sync.py

```diff
@@ -378,8 +378,8 @@ def compare_rescourcedef(self):
 
             file = os.path.join(self.momB.pbs_conf['PBS_HOME'], 'mom_priv',
                                 'hooks', 'resourcedef')
-            momret = self.momB.du.cat(self.momB.hostname, file, logerr=False,
-                                      sudo=True)
+            momret = self.momB.cat(self.momB.hostname, file, logerr=False,
+                                   sudo=True)
```

Comment:
As above, I don't see the purpose of self.mom.cat() unless you remove the hostname param.  I still don't really see the purpose then, but it at least makes a little more sense.  Doing self.mom.cat(some_nonmom_hostname, ...) makes no sense to me.